### DISCUSSION TITLE
Suoer Key

### DISCUSSION THREAD

**Discussion by oppiet30** [regolith-linux/regolith-desktop#853] [General]
What is this super key that you guys speak of? There is no super key on anyone's keyboard. You should change the text on your website to show what this key is.

**Comment by cfsmp3** [comment]
> What is this super key that you guys speak of? There is no super key on anyone's keyboard. You should change the text on your website to show what this key is.


It's the one with the penguin.

![image](https://github.com/regolith-linux/regolith-desktop/assets/5949913/5aca3e27-0925-4575-9444-b1eef05e6eb2)

**Comment by jjsearle** [comment]
It is what you probably call the 'Windows' key.
